pkg/clockcache/cache_bench_test.go

```diff
@@ -37,6 +37,168 @@ func BenchmarkStringCache(b *testing.B) {
 	}
 }
 
+type Key struct {
+	index int64
+	value string
+}
+
+type Value struct {
+	name, value string
+}
+
+// tests cache speed and memory usage for struct key and value
+func BenchmarkStructKeyValueCacheEviction(b *testing.B) {
```

Comment:
I saw that for the other benchmarks we do 

```
	for _, n := range []int{500, 5000, 50000} {
		b.Run(fmt.Sprintf("%d", n), func(b *testing.B) {
```

Is there a reason why you are not doing it like this for these benchmarks?